### ISSUE TITLE
20bn website is not available, where can I download data?

### ISSUE THREAD

**Issue by JiaMingLin** [dmlc/gluon-cv#1694]
Hi,
I need to download the something-to-something and jester datasets. But the 20bn website "[https://20bn.com](https://20bn.com)" are not available for weeks, the error message is "503 Service Temporarily Unavailable".

I have already downloaded the video data of something-to-something v2, and I need the label dataset. For the Jester, I need both video and label data. Can someone share me the download link?

Please let me know if this post is not appropriate here, I will close it immediately.

Thanks.

**Comment by mehdydousty** [comment]
> Hi,
> I need to download the something-to-something and jester datasets. But the 20bn website "https://20bn.com" are not available for weeks, the error message is "503 Service Temporarily Unavailable".
> 
> I have already downloaded the video data of something-to-something v2, and I need the label dataset. For the Jester, I need both video and label data. Can someone share me the download link?
> 
> Please let me know if this post is not appropriate here, I will close it immediately.
> 
> Thanks.

Hi Jia,
I am having the same issue that you are facing now. 20bn website is not available. Have you been able to download the data?

**Comment by sneheshs** [comment]
+1 for accessing the Jester dataset

**Comment by nesquiq** [comment]
Did someone find any alternative available download link?

**Comment by 16lemoing** [comment]
Hi, it is now available here:
https://developer.qualcomm.com/software/ai-datasets/jester
https://developer.qualcomm.com/software/ai-datasets/something-something

**Comment by sneheshs** [comment]
> Hi, it is now available here: https://developer.qualcomm.com/software/ai-datasets/jester https://developer.qualcomm.com/software/ai-datasets/something-something

Thank you!!!

**Comment by nesquiq** [comment]
> Hi, it is now available here: https://developer.qualcomm.com/software/ai-datasets/jester https://developer.qualcomm.com/software/ai-datasets/something-something

Thanks a lot for noticing!!!

**Comment by JiaMingLin** [comment]
Thanks

**Comment by dy1998** [comment]
> Hi, it is now available here: https://developer.qualcomm.com/software/ai-datasets/jester https://developer.qualcomm.com/software/ai-datasets/something-something

Thanks for your kindly notice!

**Comment by zjbthomas** [comment]
Can anybody successfully accept the "Product Kit License Agreement" in the following page?
![Snipaste_2022-02-08_17-20-28](https://user-images.githubusercontent.com/14356407/153103802-1271db22-4a8b-4d38-96e8-ab844020853c.png)


For me the page always refreshes when I click on Accept...
Tried different browsers, computers, and even networks...

**Comment by shashank2111** [comment]
Yes this issue is persisting. The accept button doesn't do any thing. The page gets refreshed.
Can anybody guide on this ?

Thank you.

**Comment by seunghyunbaik** [comment]
I mailed qualcomm and solved this problems!

**Comment by Jeba-create** [comment]
I have downloaded the something-somethingv2 dataset from here (https://developer.qualcomm.com/software/ai-datasets/something-something), but I could extract the videos only from the first folder using the command `cat` 20bn-something-something-v2-?? | tar zx`.

Please assist

**Comment by Jeba-create** [comment]
Hi Mohit,

You can download the dataset from
https://developer.qualcomm.com/software/ai-datasets/something-something.

To extract the videos, follow the following procedure

   1. unzip 20bn-something-something-v2-\??.zip
   2. cat 20bn-something-something-v2-?? | tar -xvzf -


 Please let me if this helps to solve your issue...

**Comment by keshara2032** [comment]
Hi, I need some help with this dataset. After downloading and extracting the dataset, I can not seem to find the annotation files below. 

something-something-v2-labels.json               9 KB
something-something-v2-train.json               26 MB
something-something-v2-validation.json         3.7 MB
something-something-v2-test.json               448 KB

Apologies if I am missing something but all I see is the videos. Thank you so much!

**Comment by tiendzung** [comment]
@keshara2032 Did you find the annotation files? I met the same issues. If you got it please send me. Thanks you

**Comment by Nojahhh** [comment]
When I download them I don't get a file extention, its just two files with type "FILE" named 20bn-something-something-v2-00 and 20bn-something-something-v2-01.
Tried to rename them with .zip afterwards but when I try to extract them I'm just getting a file type error.

How did you manage to extract them @keshara2032? Are you on windows or linux?

**Comment by keshara2032** [comment]
@Nojahhh Yep, I had the same issue. What i did was the following on Linux. Make sure when you execute this only those two files are there. Hope this works for you.
```cat 20bn-something-something-v2-?? | tar zx```

**Comment by Nojahhh** [comment]
> @Nojahhh Yep, I had the same issue. What i did was the following on Linux. Make sure when you execute this only those two files are there. Hope this works for you. `cat 20bn-something-something-v2-?? | tar zx`

@keshara2032 Thank you for replying, I managed to solve it using python and now I have my concatenated .tar.gz file.

In regards to your issue, if you did not manage to find those file you can download them from this link:
[Package labels](https://softwarecenter.qualcomm.com/api/download/software/dataset/AIDataset/Something-Something-V2/20bn-something-something-download-package-labels.zip)

Unzip that and you will have your json files but they are not names something-something-v2-x.json but rather just x.json.

Hope that helps you!

**Comment by keshara2032** [comment]
@Nojahhh Thank you so much for the labels!! I couldnt find them and this is really helpful. I am glad you solved the zip issue.

**Comment by t-kaichi** [comment]
> [@keshara2032](https://github.com/keshara2032) Thank you for replying, I managed to solve it using python and now I have my concatenated .tar.gz file.
> 
> In regards to your issue, if you did not manage to find those file you can download them from this link: [Package labels](https://softwarecenter.qualcomm.com/api/download/software/dataset/AIDataset/Something-Something-V2/20bn-something-something-download-package-labels.zip)
> 
> Unzip that and you will have your json files but they are not names something-something-v2-x.json but rather just x.json.
> 
> Hope that helps you!

@Nojahhh 
How did you solve the problem of extracting 20bn-something-something-v2-?? files using Python?
